wdl-engine/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Added
 
 * Implemented remote file localization for task execution (#[386](https://github.com/stjude-rust-labs/wdl/pull/386)).
+* Added `chown` workaround to `docker backend` after task execution ([#379](https://github.com/stjude-rust-labs/wdl/pull/379)).
```

Comment:
This should be under `Fixed` and should mention the actual issue being addressed (users not having permissions for files created by a root docker user). I don't think `chown` needs to be mentioned at all in the CHANGELOG (implementation detail that users reading the changelog don't need to care about).